js-miniapp-bridge/src/android/android-bridge.ts

```diff
@@ -20,6 +20,10 @@ class AndroidExcecutor implements PlatformExecutor {
       JSON.stringify({ action, param, id: callback.id })
     );
   }
+
+  getPlatform(): string {
+    return 'Android';
```

Comment:
Can we make an `enum` for this? So we could use something like `Platform.ANDROID` here.